src/pages/index.js

```diff
@@ -102,6 +158,12 @@ function closeModal(modal) {
   }
 }
 
+function newCard(item, method ="prepend") {
```

Comment:
Function names should start with a **verb** in infinitive (there is such a point in the checklist). It’s common in coding to do it so that functions could `speak` for themselves about what they do. **Please, correct it everywhere in the code**


`createNewCard`